RUN_uncor.m

```diff
@@ -0,0 +1,77 @@
+% Copyright 2008 - 2021, MIT Lincoln Laboratory
+% SPDX-License-Identifier: BSD-2-Clause
+
+%% Instruction
+% This script demonstrates how to use the UncorEncounterModel class to
+% draw samples from an uncorrelated encounter model and 
+% generate independent aircraft tracks by rejection sampling the model.
+% The default model parameter file is for a conventional aircraft. If a
+% different model is used, the start distribution needs to be updated.
+
+%% Startup script
+    file_startup = [getenv('AEM_DIR_BAYES') filesep 'startup_bayes.m'];
+run(file_startup);
+
+%% Inputs
+file_name='uncor_1200only_fwse_v1p2.txt';
+
+if contains (file_name, '.txt')
```

Comment:
I recommend creating a new file titled `RUN_uncor_CA` in the `code\matlab` directory and streamline this new file to load only the new Canadian model.

Reply:
Makes sense, will do!